src/dali/transaction_resolver.py

```diff
@@ -231,62 +231,72 @@ def resolve_transactions(
     unique_id_2_transactions: Dict[AssetAndUniqueId, List[AbstractTransaction]] = {}
     transaction: AbstractTransaction
 
-    for transaction in transactions:
```

Comment:
AFAICT the main addition here is the try catch, but the combination of new indentation + formatting changes (like in line 250/251) throws off the diff algorithm and we get a large diff as a result.

Since the formatting changes are not relevant to this PR and they cause the diff to be unnecessarily large, could you undo the formatting changes and only leave in the logic you want to change? If you have formatting changes to propose it's probably better to do it in a different PR. 

The reason I'm more conservative than usual with changes to this file is that the transaction resolver is a central element of the system and I want to make sure I review all logic changes thoroughly (and mixing in formatting changes makes that a bit harder).

Reply:
This is a major issue for me. I need to interrupt the process and save the cache file. Basically, it still takes 10-12 hours to make a full pull on my laptop which I need to unplug and move with me. Saving the cache file now takes a significant amount of time so it is often interrupted if I save periodically. I need to save on exit in order to test.

Since I'm under a deadline (I need to file taxes). I'll have to unfortunately keep this in place for now while I finish working. I'll try to remember to roll it back before submitting.

Reply:
To clarify further: I am not requesting to roll back the new functionality (saving the cache), but only the formatting changes (such as those on line 250/251). The reason is that new indentation + formatting changes seems to throw off the diff algorithm which generated a large diff unnecessarily (and that is harder to review). I do understand you're on a deadline though: there is no rush.

Reply:
OK, these formatting changes were because the line was too long. Should I just disable that check for now?

Reply:
No problem: on second thought it's better to just follow the indications of static analysis without suppressing anything. In the end I diffed this file using Emacs and that diff was very easy to read.